### DISCUSSION TITLE
Specific example

### DISCUSSION THREAD

**Discussion by sheridangis** [bertrandmartel/tableau-scraping#11] [Q&A] [answered]
In your sample you listed
https://public.tableau.com/views/OregonCOVID-19CasesbyZIPCode-SummaryTable/CasesbyZIPCodeSummaryTable

I have a table similar, wondering exactly which set of codes you used, different columns but similar layout.

So far I have been unsuccessful in running the code with this example.

My particular site is this one: 
https://public.tableau.com/profile/melissa.taylor#!/vizhome/EpiCOVIDtest/CountyNumbersTable

**Comment by bertrandmartel** [comment, marked as answer]
@sheridangis Hello, in your case the url would be https://public.tableau.com/views/EpiCOVIDtest/CountyNumbersTable

There was a bug #9 regarding the datatypes, if you upgrade to v0.1.8, the following works fine:

```python
from tableauscraper import TableauScraper as TS

url = "https://public.tableau.com/views/EpiCOVIDtest/CountyNumbersTable"

ts = TS()
ts.loads(url)

workbook = ts.getWorkbook()

for t in workbook.worksheets:
    print(f"worksheet name : {t.name}") 
    print(t.data) 
```
https://replit.com/@bertrandmartel/TableauCovid19Wyoming

**Reply by sheridangis** [reply]
Thanks for the link update.  I will see what I can do with that but sounds like you might still be working on something.  I know nothing about Tableau and still have lots to learn about python.

**Reply by bertrandmartel** [reply]
@sheridangis please see the updated comment above
